mc/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=mc
+pkg_origin=core
+pkg_version=4.8.18
+pkg_description="Midnight Commander."
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gnu')
```

Comment:
This should follow the [SPDX format](https://spdx.org/licenses/) and be "GPL-3.0"